gui/main.py

```diff
@@ -2,17 +2,13 @@
 from PyQt5.QtWidgets import QApplication
 
 from gui.configuration_provider import ConfigurationProvider
-from gui.functionality.idact_app import IdactApp
+from gui.functionality.main_window import MainWindow
 
 if __name__ == '__main__':
-    debug = False
```

Comment:
Why have you deleted the `debug` mode? Is there a way to stop showing all logs only in log_window? Or maybe we ultimately want co create something like `idact_gui_log` to copy all information from he widget to file? So why deleting this here?

Reply:
I didn't see the point in allowing user to run application in 'debug' mode. I would say that application is for user who don't want to change anything in application. When programmer wants logs back on the console, it can be easily done by commenting two lines of code.

If you don't agree or see how it would help we can discuss on it.

Reply:
I see your point but I think user still should have a way to see logs(errors of the application) when it could close itself rapidly because of an error. But I think we must just do it in another PR.